pretender.js

```diff
@@ -333,13 +335,15 @@ Pretender.prototype = {
   passthrough: PASSTHROUGH,
   checkPassthrough: function checkPassthrough(request) {
     var verb = request.method.toUpperCase();
-
     var path = parseURL(request.url).fullpath;
-
-    verb = verb.toUpperCase();
-
     var recognized = this.hosts.forURL(request.url)[verb].recognize(path);
     var match = recognized && recognized[0];
+
+    if (!match && this.forcePassthrough) {
```

Comment:
if we are forcing passthrough, why does the result of `match` matter?

Reply:
Also when forcing passthrough, why do we register a new route? My intuition would be to immediately invoke:

```js
this.passthroughRequests.push(request);
this.passthroughRequest(verb, path, request);
```

Reply:
just to avoid possible duplicate, I look at match, thats why i though register the route was a good idea, I give a try to your solution and come back to you!